src/node/internal/zlib.d.ts

```diff
@@ -2,6 +2,21 @@ import { owner_symbol, type Zlib } from 'node-internal:internal_zlib_base';
 
 export function crc32(data: ArrayBufferView, value: number): number;
 
+type CompressCallback<ErrT, BufT> = (error: ErrT | null, result: BufT) => void;
```

Comment:
I don't understand typescript. How does this work without an export?

```suggestion
export type CompressCallback<ErrT, BufT> = (error: ErrT | null, result: BufT) => void;
```

Reply:
Apparently TS allows this: https://github.com/microsoft/TypeScript/issues/38592

Reply:
(but I will fix)